lib/plugins/TaskRunner/Drupal.js

```diff
@@ -31,7 +32,8 @@ module.exports = class Drupal extends require('./Script') {
     options.secrets = [
     ];
 
-    var subDirectory = options.subDirectory || 'docroot';
+    // Allow for subdirectory to be explicitly set to "" without being overwritten for being falsy.
+    this.subDirectory = ((options.subDirectory || options.subDirectory === "") ? options.subDirectory : 'docroot');
```

Comment:
this can actually be `this.subDirectory = options.subDirectory || 'docroot';`, I don't think there's any reason to test for an empty string is there? If it's undefined we don't use it - if it is defined an empty string is just one of many ways it could be wrong.